src/sql/src/plan.rs

```diff
@@ -467,7 +466,7 @@ pub enum SourceSinkClusterConfig {
     /// Use an existing cluster.
     Existing {
         /// The ID of the cluster to use.
-        id: StorageInstanceId,
+        id: ClusterId,
```

Comment:
`ClusterId` is a type alias for `StorageInstanceId`. I just thought it was more appropriate here since that what we use for all clusters in the planner/adapter layer.